buildenv/jenkins/openjdk_tests

```diff
@@ -150,136 +150,135 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
         parallel testJobs
     }
 } else {
-    if (PLATFORM_MAP.containsKey(params.PLATFORM)) {
-        SPEC = PLATFORM_MAP[params.PLATFORM]["SPEC"]
-        LABEL = params.LABEL ? params.LABEL : PLATFORM_MAP[params.PLATFORM]["LABEL"]
+    if (!PLATFORM_MAP.containsKey(params.PLATFORM)) {
```

Comment:
This is one example of an `if` that I negated. It is much easier for me to read "if not this then exit" compared to "if this .... 130 lines later ... else exit". I know the plain diff on GH is tough to read, so if you look at only the first commit, and select "Hide whitespace changes", you'll see that's how the change works.